epoxy-processortest/src/test/java/com/airbnb/epoxy/DataBindingModelTest.java

```diff
@@ -96,4 +99,23 @@ public void testSimpleModelNoValidation() {
         .and()
         .generatesSources(generatedModel);
   }
+
+  @Test
+  public void testFullyGeneratedModel() {
+    JavaFileObject packageInfo = JavaFileObjects
```

Comment:
possibly define the package-info file as a field in this class (like R and BR), since `package-info` is otherwise very vague and might be mixed up with other tests.